frontend/src/components/RecentOperations/OperationTypeColumn.tsx

```diff
@@ -0,0 +1,203 @@
+import React from "react";
+import BigNumber from "bignumber.js";
+
+import { FetchLastOperationsActionResponse } from "types";
+import { AssetLink } from "components/AssetLink";
+import { AccountBadge } from "components/AccountBadge";
+import { LiquidityPoolBadge } from "components/LiquidPoolBadge";
+
+interface OperationTypeColumnProps {
+  operation: FetchLastOperationsActionResponse;
+  horizonURL: string;
+}
+
+interface FormatAccountProps {
+  amount: number;
+  assetType: string;
+  assetCode?: string;
+  assetIssuer?: string;
+}
+
+export const OperationTypeColumn: React.FC<OperationTypeColumnProps> = ({
+  operation,
+  horizonURL,
+}) => {
+  const formatAccount = ({
```

Comment:
this should be formatAmount no?
```suggestion
  const formatAmount = ({
```